src/workflow/ArcanistStackWorkflow.php

```diff
@@ -854,19 +866,9 @@ public function didPush() {
     }
     $this->askForRepositoryUpdate();
 
-    $mark_workflow = $this->buildChildWorkflow(
-      'close-revision',
-      array(
-        '--finalize',
-        '--quiet',
-        $this->revision['id'],
-      ));
-    $mark_workflow->run();
-    // UBER CODE
```

Comment:
Closing revision in stacked changes should be left to Phabricator commit parser, though it is possible to collect all revisions which were affected...